src/main/java/com/minelittlepony/PonyManager.java

```diff
@@ -19,108 +25,160 @@
 import java.util.UUID;
 import java.util.stream.Collectors;
 
+/**
+ * The PonyManager is responsible for reading and recoding all the pony data associated with an entity of skin.
+ *
+ */
 public class PonyManager implements IResourceManagerReloadListener {
 
     public static final ResourceLocation STEVE = new ResourceLocation("minelittlepony", "textures/entity/steve_pony.png");
     public static final ResourceLocation ALEX = new ResourceLocation("minelittlepony", "textures/entity/alex_pony.png");
-
-    private static final ResourceLocation BGPONIES_JSON = new ResourceLocation("minelittlepony", "textures/entity/pony/bgponies.json");
+    public static final ResourceLocation BGPONIES_JSON = new ResourceLocation("minelittlepony", "textures/entity/pony/bgponies.json");
```

Comment:
Why does this need to be public?

Reply:
Why did you leave STEVE and ALEX public? Regardless, I always prefer to leave constants lile this public as a nice gesture to other developers.